docs/fields/upload.mdx

```diff
@@ -78,12 +78,15 @@ export const ExampleCollection: CollectionConfig = {
 
 ### Filtering upload options
 
-Options can be dynamically limited by supplying a [query constraint](/docs/queries/overview), which will be used both for validating input and filtering available uploads in the UI.
+Options can be dynamically limited by supplying a [query constraint](/docs/queries/overview), which will be used both
+for validating input and filtering available uploads in the UI.
 
-The `filterOptions` property can either be a `Where` query directly, or a function that returns one. When using a function, it will be called with an argument object with the following properties:
+The `filterOptions` property can either be a `Where` query directly, or a function that returning `null` to not alter
```

Comment:
1. Typo: that returning => that returns
2. The way it's typed, it's not a function that returns null. It's just the value null - no function

Reply:
1. makes sense
2. I think you're right. This is not correct on main currently, but I can fix the type to match the docs.